docs/content/concepts/partitions-schedules-sensors/asset-sensors.mdx

```diff
@@ -218,53 +242,18 @@ def repository_1():
 
 This sensor will materialize `asset_b` whenever any of its parents materialize, and it will materialize `asset_c` whenever any of its parents materialize.
 
-## Examples
+### Freshness-based scheduling <Experimental />
 
-### Monitoring daily assets to materialize a weekly asset
```

Comment:
I'd like to keep this example since users have asked for it

Reply:
Hm can they just use build_asset_reconciliation_sensor in that case? (that's why I removed it but I'll admit I'm not the most familiar w/ this part of the world)

Reply:
Code snippet of what the "AND" case would look like (I tested to confirm this works):
```
@multi_asset_sensor(
    asset_keys=[AssetKey("upstream_daily_1")], job=weekly_asset_job, minimum_interval_seconds=5
)
def trigger_weekly_asset_from_daily_asset(context):
    run_requests = []
    materializations_by_partition = context.latest_materialization_records_by_partition(
        AssetKey("upstream_daily_1")
    )

    materializations_by_downstream_weekly_partition = defaultdict(list)
    # Get all corresponding weekly partitions for any materialized daily partitions
    for partition, materialization in materializations_by_partition.items():
        # Get the corresponding weekly partition, None if no weekly partition exists
        downstream_partitions = context.get_downstream_partition_keys(
            partition,
            from_asset_key=AssetKey("upstream_daily_1"),
            to_asset_key=AssetKey("downstream_weekly_asset"),
        )
        if downstream_partitions:
            # A daily partition can only have at most one corresponding weekly partition
            materializations_by_downstream_weekly_partition[downstream_partitions[0]].append(
                materialization
            )

    for (
        downstream_weekly_partition,
        daily_materializations,
    ) in materializations_by_downstream_weekly_partition.items():
        daily_partitions_in_week = context.get_downstream_partition_keys(
            downstream_weekly_partition,
            from_asset_key=AssetKey("downstream_weekly_asset"),
            to_asset_key=AssetKey("upstream_daily_1"),
        )
        if all(
            [
                daily_partition in materializations_by_partition
                for daily_partition in daily_partitions_in_week
            ]
        ):
            run_requests.append(
                weekly_asset_job.run_request_for_partition(downstream_weekly_partition)
            )
            for daily_materialization in daily_materializations:
                # Advance the cursor so we only check event log records past the cursor
                context.advance_cursor({AssetKey("upstream_daily_1"): daily_materialization})
    return run_requests
```

Reply:
ah perfect -- thanks for the code snippet. I'll add this back in and add a note about how this relates to the reconciliation sensor logic

Reply:
I actually ended up keeping the original snippet and just explaining that it did the same thing as the build_asset_reconciliation_sensor, and was more of a jumping-off point